lib/internal/crypto/diffiehellman.js

```diff
@@ -122,8 +122,15 @@ function DiffieHellman(sizeOrKey, keyEncoding, generator, genEncoding) {
     generator = DH_GENERATOR;
   else if (typeof generator === 'number')
     validateInt32(generator, 'generator');
-  else
+  else if (generator !== true)
     generator = toBuf(generator, genEncoding);
+  else
```

Comment:
If we're going to add multiple lines below, can we please add braces to the `if`/`else if`s/`else` to make things more readable?